config-reloader v0.47.0 crashes

After v0.47.0 config-reloader crashes with following error:


```
/app flag redefined: tlsCertFile
panic: /app flag redefined: tlsCertFile

goroutine 1 [running]:
flag.(*FlagSet).Var(0x400017a000, {0x14f1a78, 0x4000386140}, {0x123cc6a, 0xb}, {0x12a564a, 0x57})
	/usr/local/go/src/flag/flag.go:1028 +0x2cc
flag.(*FlagSet).StringVar(...)
	/usr/local/go/src/flag/flag.go:879
flag.(*FlagSet).String(0x400017a000, {0x123cc6a, 0xb}, {0x0, 0x0}, {0x12a564a, 0x57})
	/usr/local/go/src/flag/flag.go:892 +0x98
flag.String(...)
	/usr/local/go/src/flag/flag.go:899
main.init()
	/workspace/cmd/config-reloader/main.go:61 +0x538
```

 Bug was introduced at f3efa3f1bdd2b57d4fc6962e15e3c0f7ed3faff7

--- a/cmd/config-reloader/main.go
+++ b/cmd/config-reloader/main.go
@@ -56,15 +56,15 @@ var (
 	webhookMethod = flag.String(
 		"webhook-method", "GET", "the HTTP method url to use to send the webhook")
 
-	tlsCaFile = flag.String("tlsCAFile", "",
+	tlsCaFile = flag.String("reload.tlsCAFile", "",
 		"Optional path to client-side TLS CA file to use when connecting to -reload-url")
-	tlsCertFile = flag.String("tlsCertFile", "",
+	tlsCertFile = flag.String("reload.tlsCertFile", "",
 		"Optional path to client-side TLS certificate file to use when connecting to -reload-url")
-	tlsKeyFile = flag.String("tlsKeyFile", "",
+	tlsKeyFile = flag.String("reload.tlsKeyFile", "",
 		"Optional path to client-side TLS key file to use when connecting to -reload-url")
-	tlsServerName = flag.String("tlsServerName", "",
+	tlsServerName = flag.String("reload.tlsServerName", "",
 		"Optional TLS server name to use for connections to -realod-url.")
-	tlsInsecureSkipVerify = flag.Bool("tlsInsecureSkipVerify", true,
+	tlsInsecureSkipVerify = flag.Bool("reload.tlsInsecureSkipVerify", true,
 		"Whether to skip tls verification when connecting to -reload-url")
 )
 

--- a/docs/CHANGELOG.md
+++ b/docs/CHANGELOG.md
@@ -12,6 +12,7 @@ aliases:
 ---
 
 - fixed Prometheus scrape config metricsPath conversion. See [this issue](https://github.com/VictoriaMetrics/operator/issues/1073)
+- [config-reloader](https://docs.victoriametrics.com/operator/): Added `reload` prefix to all config-reloader `tls*` flags to avoid collision with flags from external package. See [this issue](https://github.com/VictoriaMetrics/operator/issues/1072)
 
 ## [v0.47.0](https://github.com/VictoriaMetrics/operator/releases/tag/v0.47.0) - 15 Aug 2024
 

